include/picolibrary/microchip/megaavr0/peripheral/clkctrl.h

```diff
@@ -246,6 +376,62 @@ class CLKCTRL {
         auto operator=( MCLKSTATUS const & ) = delete;
 
         using Register<std::uint8_t>::operator=;
+
+        /**
+         * \brief Check if the clock source is changing.
+         *
+         * \return true if the clock source is changing.
+         * \return false if the clock source is not changing.
+         */
+        auto source_changing() const noexcept -> bool
+        {
+            return *this & Mask::SOSC;
+        }
+
+        /**
+         * \brief Check if the internal 16/20 MHz oscillator is stable.
+         *
+         * \return true if the internal 16/20 MHz oscillator is stable.
+         * \return false if the internal 16/20 MHz oscillator is not stable.
+         */
+        auto internal_16_20_mhz_oscillator_stable() const noexcept -> bool
+        {
+            return *this & Mask::OSC20MS;
+        }
+
+        /**
+         * \brief Check if the internal 32.768 kHz ultra low-power oscillator is stable.
+         *
+         * \return true if the internal 32.768 kHz ultra low-power oscillator is stable.
+         * \return false if the internal 32.768 kHz ultra low-power oscillator is not
+         *         stable.
+         */
+        auto internal_32_768_khz_ultra_low_power_oscillator_stable() const noexcept -> bool
+        {
+            return *this & Mask::OSC32KS;
+        }
+
+        /**
+         * \brief Check if the external 32.768 kHz crystal oscillator is stable.
+         *
+         * \return true if the external 32.768 kHz crystal oscillator is stable.
+         * \return false if the external 32.768 kHz crystal oscillator is not stable.
+         */
+        auto external_32_768_khz_crystal_oscillator_stable() const noexcept -> bool
+        {
+            return *this & Mask::XOSC32KS;
+        }
+
+        /**
+         * \brief Check if the external clock is stable.
+         *
+         * \return true if the external clock is stable.
+         * \return false if the external clock is not stable.
+         */
+        auto external_clock_stable() const noexcept
```

Comment:
Deduced return type is not correct.
```suggestion
        auto external_clock_stable() const noexcept -> bool
```